pyterrier/__init__.py

```diff
@@ -15,17 +11,22 @@
 
 HOME_DIR = None
 
-
-def init(version=None, mem=None, packages=[], jvm_opts=[], redirect_io=True, logging='WARN', home_dir=None):
+def init(version=None, mem=4096, packages=[], jvm_opts=[], redirect_io=True, logging='WARN', home_dir=None):
     """
     Function necessary to be called before Terrier classes and methods can be used.
     Loads the Terrier.jar file and imports classes. Also finds the correct version of Terrier to download if no version is specified.
 
     Args:
         version(str): Which version of Terrier to download. Default=None.
             If None, find the newest Terrier version in maven and download it.
-        mem: Maximum memory allocated for java heap in MB. Default=4096.
-        packages: Extra .jar files to load.
+        mem(str): Maximum memory allocated for java heap in MB. Default=4096.
+        packages(list(str)): Extra .jar files to load. Default=[].
```

Comment:
this should refer to Maven package coordinates, not jar files.

Reply:
Ok. I just left the message as written.

Reply:
While we are here, we should improve things!